src/OrleansCounterControl/Microsoft.Orleans.CounterControl.targets

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="15.0" DefaultTargets="Build">
```

Comment:
but the thing is that we DO support our nugets in VS2015. We just stopped supporting opening the Orleans solution itself in VS2015. This file file ends up in the nuget package that users consume.